Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSendButton.swift

```diff
@@ -5,19 +5,25 @@
 import StreamChat
 import UIKit
 
+/// Button used for final sending or confirming editing messages in ComposerView.
 public typealias ChatMessageComposerSendButton = _ChatMessageComposerSendButton<NoExtraData>
 
+/// Button used for final sending or confirming editing messages in ComposerView.
 open class _ChatMessageComposerSendButton<ExtraData: ExtraDataTypes>: _ComposerButton<ExtraData> {
+    /// Mode for the button
+    /// - new: When user wants to send new message.
+    /// - edit: When user edited his message and confirm changes.
     public enum Mode {
-        case new, edit
+        case new
+        case edit
```

Comment:
can we be more expressive here? Maybe `Mode` -> `MessageKind` -> `newMessage` and `editedMessage`